fax/loop.py

```diff
@@ -104,20 +103,19 @@ def fixed_point_iteration(init_x, func, convergence_test, max_iter,
 
     def cond(args):
         i, x_new, x_old = args
+        x_new, x_old = get_params(x_new), get_params(x_old)
         converged = convergence_test(x_new, x_old)
 
         if max_iter is not None:
             converged = converged | (max_iter <= i)
         return np.logical_not(converged)
 
     def body(args):
-        i, x_new, _ = args
-        i_new, x_new, x_old = unrolled(i, x_new, func, batched_iter_size,
-                                       return_last_two=True)
+        i, x_new, _x_old = args
```

Comment:
Same comment as above: would be good not to interfere with the formatting too much.

Reply:
reverted